PatientWebApplication/HealthClinic.DAL/Service/MessageService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using HealthClinic.CL.Adapters;
+using HealthClinic.CL.DbContextModel;
+using HealthClinic.CL.Dtos;
+using HealthClinic.CL.Model.ActionsAndBenefits;
+using HealthClinic.CL.Repository;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HealthClinic.CL.Service
+{
+    public class MessageService:IMessageService
+    {
+        private IMessageRepository MessageRepository { get; set; }
+        public MessageService()
+        {
+            MessageRepository = new MessageRepository();
+        }
+
+        public MessageService(IMessageRepository messageRepository)
+        {
+            MessageRepository = messageRepository;
+        }
+
+        public Message Create(MessageDTO dto)
+        {
+            Message message = MessageAdapter.MessageDTOtoMessage(dto);
+            return MessageRepository.Create(message);

```

Comment:
Write this in form:
"return MessageRepository.Create(MessageAdapter.MessageDTOtoMessage(dto));"